src/masternodes/mn_rpc.cpp

```diff
@@ -2902,6 +2913,10 @@ UniValue listaccounthistory(const JSONRPCRequest& request) {
             if (height > startKey.blockHeight || (depth <= startKey.blockHeight && (height < startKey.blockHeight - depth)))
                 return true; // continue
 
+            if (txType && txType != category) {
```

Comment:
Should explicitly compare with `CustomTxType::None` as `CustomTxType::None != txType`

Reply:
If i get it above by `CustomTxType` i should cast here, category is unsigned char

Reply:
Yes, but at least the first element need to do comparison.

Reply:
We know `None` is 0, it's not going to change :) I'll do the change.